src/main/java/frc/robot/ports.java

```diff
@@ -0,0 +1,15 @@
+package frc.robot;
+
+public class ports {
+    public static class Drivetrain{
+        public static final int RIGHT_SLAVE = 0;
+        public static final int RIGHT_MASTER = 1;

```

Comment:
Please move the Master above the slave, and also replace the ports between the motors, and also replace the ports between the motors.